prog/postgres/postgres_resource_nexus.rb

```diff
@@ -48,9 +48,12 @@ def self.assemble(project_id:, location:, name:, target_vm_size:, target_storage
       )
       postgres_resource.associate_with_project(project)
 
+      private_subnet_id = Prog::Vnet::SubnetNexus.assemble(Config.postgres_service_project_id, name: "#{postgres_resource.ubid}-subnet", location: location).id
+      postgres_resource.update(private_subnet_id: private_subnet_id)
+
       PostgresFirewallRule.create_with_id(postgres_resource_id: postgres_resource.id, cidr: "0.0.0.0/0")
 
-      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative_at: Time.now)
+      Prog::Postgres::PostgresServerNexus.assemble(resource_id: postgres_resource.id, timeline_id: timeline_id, timeline_access: timeline_access, representative_at: Time.now, private_subnet_id: private_subnet_id)
```

Comment:
You already pass the resource to it. IMHO, it can use resource's private subnet directly. Do we have any case that we want to give another private subnet?